src/mds/SnapRealm.h

```diff
@@ -129,7 +129,7 @@ struct SnapRealm {
   std::set<SnapRealm*> open_children;    // active children that are currently open
 
   elist<CInode*> inodes_with_caps;             // for efficient realm splits
-  std::map<client_t, xlist<Capability*>* > client_caps;   // to identify clients who need snap notifications
+  std::map<client_t, elist<Capability*> > client_caps;   // to identify clients who need snap notifications
```

Comment:
This is a nice saving. Just for information sake, here is the size difference:

> 16   elist<>::item
> 32   xlist<>::item

This is from a sample MDS log (main branch build) that dumps struct sizes on start. 👍